README.md

```diff
@@ -137,6 +137,12 @@ You can deploy this project using:
 - Twitter: [@twitterhandle](https://twitter.com/mwovi_peter)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/peter-mwovi-57141a179/)
 
+👤 **Juan Carlos Muñoz**
+
+- GitHub: [@githubhandle](https://github.com/jcmunav63)
+- Twitter: [@twitterhandle](https://twitter.com/jcmunav63)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/juan-carlos-mu%C3%B1oz-navarrete-5a15b6276/)
+
```

Comment:
Great job on adding the second author to your project! It's important to acknowledge all contributors who have made significant contributions to the project. However, upon reviewing your project's commit history, I noticed that there is another person who has also made contributions.

I kindly request you to include this additional contributor in the author section of your project to give proper credit to everyone involved. It demonstrates transparency and recognizes the collaborative effort put into the project.
![image](https://github.com/peter-mwovi/portfolio_setup/assets/55840999/66b28075-123e-496c-8d06-ab057a602939)

By including all the contributors in the author section, you will accurately represent the collective work and contributions made by the team. Keep up the good work and ensure that credit is given where it's due! 👏👥

Reply:
So, the other contributor is still me. I happen to have a separate github account and I couldn't set this one up properly in my local machine so everytime I make a contribution to this one, it marks that it is contributed from my other profile.